app/lib/isNotEnglishPostcode.js

```diff
@@ -0,0 +1,8 @@
+// http://www.roblocher.com/technotes/uk-postareas.html
+const excludedRegEx = /^(AB|BT|CF|DD|EH|FK|G|GY|HS|IM|IV|JE|KA|KW|KY|ML|PA|PH|SA|ZE)\d+/i;
```

Comment:
I'd argue that this is premature optimisation and that we should rely on whatever reverse geocoding service we are using to tell us which country the post code is in.

There are a number of ways we could optimise should we identify a need to; this would be one but another could be to have our own instance of postcodeio running in a docker image in our infrastructure.